packages/rooks/tsconfig.json

```diff
@@ -9,7 +9,6 @@
     "module": "esnext",
     "jsx": "react",
     "target": "esnext",
-    "composite": true,
```

Comment:
I had to remove this cause I will make the rollup-plugin-typescript-paths fail. See above comment.

I feel there's no need to set it, cause I haven't seen project references support

Reply:
Got it. If we don't need it, let's scrap. 👍🏽